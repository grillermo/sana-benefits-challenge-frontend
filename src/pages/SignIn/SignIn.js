import SignInForm from '../../components/SignInForm'

const SignIn = () => {
  return(
  <div>
    <h2>
      Sign in
    </h2>

    <SignInForm />
  </div>
  )
}

export default SignIn
