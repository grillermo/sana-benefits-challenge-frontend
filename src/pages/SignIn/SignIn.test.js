import { shallow } from 'enzyme'

import SignIn from './SignIn'

describe('SignIn', () => {
  it('matches the snapshot', async () => {
    const signIn = shallow(<SignIn />)

    expect(signIn).toMatchSnapshot();
  });
})
