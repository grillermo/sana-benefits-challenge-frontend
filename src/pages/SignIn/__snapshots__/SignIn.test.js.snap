// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn matches the snapshot 1`] = `
<div>
  <h2>
    Sign in
  </h2>
  <SignInFormContainer />
</div>
`;
